torch/csrc/jit/passes/quantization.cpp

```diff
@@ -365,8 +366,12 @@ void InsertObserversHelper::insertObservers(
   script::Method method = module.get_method(method_name);
   auto graph = method.graph();
   ConstantPropagation(graph);
+  // To cleanup traced graph
+  ConstantPooling(graph);
```

Comment:
Just curious, is there any order we can follow for the optimization passes?

Reply:
not sure, looks like in graph_executor constant pooling is done before constant propagation, I'll change the order

Reply:
If we want to cleanup traced graphs, shouldn't we do it in tracing?

Reply:
Not sure why we don't do this in traced graph, cc @jamesr66a

Reply:
Was speaking with @suo yesterday. It's a pretty common request to add more "clean-up" passes to Module emission (i.e. tracing or scripting) as opposed to just in the GraphExecutor. I think this PR is good, but it would also be valid to just add these things to tracing